src/Products/SiteAccess/VirtualHostMonster.py

```diff
@@ -104,7 +104,7 @@ def set_map(self, map_text, RESPONSE=None):
                 if hostname not in host_map:
                     host_map[hostname] = {}
                 host_map[hostname][port] = pp
-            except 'LineError' as msg:
+            except Exception as msg:
```

Comment:
```suggestion
            except ValueError as msg:
```

This would restore the original behavior. (See my other comment.)